sheeprl/algos/p2e_dv1/p2e_dv1_exploration.py

```diff
@@ -835,7 +835,6 @@ def main(fabric: Fabric, cfg: Dict[str, Any]):
     envs.close()
     # task test zero-shot
     if fabric.is_global_zero and cfg.algo.run_test:
-        player.actor = actor_task.module
```

Comment:
Here you want to test the task, not the exploration. So you must replace the exploration actor with the task actor.